prometheus/types.go

```diff
@@ -13,5 +13,6 @@ type Alert struct {
 type Scrape struct {
 	ScrapePort 	int `json:"scrapePort,string,omitempty"`
 	ServiceName string `json:"serviceName"`
+	ScrapeType string
```

Comment:
I decided to do not change it at first because I thought it would not be interesting for the project, but I agree it would probably be useful, I will implement it!